packages/eas-cli/src/commands/branch/publish.ts

```diff
@@ -118,11 +118,13 @@ export default class BranchPublish extends Command {
     const assetSpinner = ora('Uploading assets...').start();
     try {
       const assets = collectAssets(inputDir!);
+      console.log('[debug] assets -', 'uploading these assets:', JSON.stringify(assets, null, 2));
       await uploadAssetsAsync(assets);
       updateInfoGroup = await buildUpdateInfoGroupAsync(assets);
       assetSpinner.succeed('Uploaded assets!');
     } catch (e) {
       assetSpinner.fail('Failed to upload assets');
+      console.log('[debug] assets -', 'failed because of', e);
```

Comment:
```suggestion
      Log.log('[debug] assets -', 'failed because of', e);
```

Reply:
Also, maybe just put this error message into the assetSpinner.fail message on the line above?

Reply:
Actually did you mean to delete these?